### DISCUSSION TITLE
Not support export const a = props.a

### DISCUSSION THREAD

**Discussion by gutenye** [mdx-js/mdx#2075] [Q&A] [answered]
Is this expected? I'd like to use `{a}` instead of `{props.a}` in mdx for short sugar syntax.

**Comment by wooorm** [comment, marked as answer]
Yes. Import/export statements are a level higher than the content. You can’t pass props to them. To illustrate, say this is `file.mdx`:

```
export const a = 1

{props.b}
```

And this is the code using it:

```js
import {default as MDXContent, a} from './path/to/file.mdx'

// `MDXContent` is a function, so sure, we could pass props to them and set `b`.
// But `a` is the value `1`, it’s already exported, we can’t change that.
```
